example/ck_tile/01_fmha/codegen/ops/fmha_fwd.py

```diff
@@ -550,7 +550,7 @@ def get_hdim_tile_size_dict(dtype : str) -> Optional[dict]:
                 (192,192) : [FmhaFwdTileSize(128, 128, 32, 192, 32,  192,  4, 1, 1,  4, 1, 1,  32, 32, 16,  32, 32, 16,   1)],
                 (256,256) : [FmhaFwdTileSize(128, 128, 32, 256, 32,  256,  4, 1, 1,  4, 1, 1,  32, 32, 16,  32, 32, 16,  -1)],
             }
-        elif dtype == 'fp8' or dtype == 'bf8':
+        elif dtype == 'fp8' or dtype == 'fp8bf16':
```

Comment:
Just curious, if we change output datatype to bf16, should we change the tile size